.github/CODEOWNERS

```
@@ -136,6 +136,8 @@
 /modules/programs/sbt.nix                             @kubukoz
 /tests/modules/programs/sbt                           @kubukoz
 
+/modules/programs/scmpuff.nix                         @cpcloud
```

Go ahead and add yourself as the codeowner for the tests as well:

```suggestion
/modules/programs/scmpuff.nix                         @cpcloud
/tests/modules/programs/scmpuff                       @cpcloud
```